user/urls.py

```diff
@@ -1,5 +1,11 @@
+"""Urls.py manages the urls for the user management
+
+Attributes
+----------
+app_name : str
+    sets the app name to call it e.g. in templates.
```

Comment:
Capitalize, I don't understand the description, please try to explain better what this attribute is.